torch/csrc/jit/serialization/pickler.cpp

```diff
@@ -310,6 +310,8 @@ void Pickler::pushStorageOfTensor(const at::Tensor& tensor) {
   // change will only be triggered on non-mobile logic
   if (get_tensor_id_ != nullptr) {
     root_key = get_tensor_id_(tensor);
+  } else {
+    root_key = c10::to_string(tensor_data_.size());
```

Comment:
Looks a good change to me. @suo and @Lilyjjo please confirm if it's good.

Reply:
It's a temporary fix and @Lilyjjo addressed it in the stack

Reply:
Rebase and there is no change in `pickler.cpp` now.